website/content/en/preview/getting-started/_index.md

```diff
@@ -13,3 +13,211 @@ Learn more about Karpenter and how to get started below.
+
+{{% script file="./scripts/step06-tag-subnets.sh" language="bash" %}}
+
+Add tags to the security groups.
+This command only tags the security groups for the first nodegroup in the cluster.
+If you have multiple nodegroups or multiple security groups you will need to decide which one Karpenter should use.
+
+{{% script file="./scripts/step07-tag-security-groups.sh" language="bash" %}}
+
+### Deploy Karpenter
+
+First set the Karpenter release you want to deploy and the IAM instance profile that is used by the existing nodes in your cluster.
+```bash
+export KARPENTER_VERSION={{< param "latest_release_version" >}}
+export IAM_INSTANCE_PROFILE_NAME=<your instance profile name>
+```
+
+We can now generate a full Karpenter deployment yaml from the helm chart.
+
+{{% script file="./scripts/step09-generate-chart.sh" language="bash" %}}
+
+Modify the following lines in the karpenter.yaml file.
+
+### Set node affinity
+
+Edit the karpenter.yaml file and find the karpenter deployment affinity rules.
+Modify the affinity so karpenter will run on one of the existing node group nodes.
+
+The rules should look something like this.
+Modify the value to match your `$NODEGROUP`.
+
+```
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: karpenter.sh/provisioner-name
+                operator: DoesNotExist
+            - matchExpressions:
+              - key: eks.amazonaws.com/nodegroup
+                operator: In
+                values:
+                - ${NODEGROUP}
+```
+
+Now that our deployment is ready we can create the karpenter namespace, create the provisioner CRD, and then deploy the rest of the karpenter resources.
+
+{{% script file="./scripts/step10-deploy.sh" language="bash" %}}
+
+## Create default provisioner
+
+We need to create a default provisioner so Karpenter knows what types of nodes we want for unscheduled workloads.
+You can refer to some of the [example provisioners](https://github.com/aws/karpenter/tree{{< githubRelRef >}}examples/provisioner) for specific needs.
+
+{{% script file="./scripts/step11-create-provisioner.sh" language="bash" %}}
+
+## Set nodeAffinity for critical workloads (optional)
+
+You may also want to set a nodeAffinity for other critical cluster workloads.
+
+Some examples are
+
+* coredns
+* metric-server
+
+You can edit them with `kubectl edit deploy ...` and you should add node affinity for your static node group instances.
+Modify the value to match your `$NODEGROUP`.
+
+```
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: eks.amazonaws.com/nodegroup
+                operator: In
+                values:
+                - ${NODEGROUP}
+```
+
+## Remove Cluster Autoscaler
```

Comment:
I wonder if it's worth including this in the getting started guide. Actually migrating from CAS requires a lot more intention/planning.